### ISSUE TITLE
Separate out title and url field

### ISSUE THREAD

**Issue by suhaschitade** [lane711/sonicjs#128]
Many times cotent creator want some superb title which can be a bit long and in other language. 
Having linked with autogenerated url system is not working if the title is in other language, Please find the screenshot

![image](https://user-images.githubusercontent.com/8472771/168761039-564d0f8f-acbd-4081-8ac5-be4de31ef60c.png)

**Comment by lane711** [comment]
Good call, will fix this by allowing the user to override the auto generate

Thanks
